### PR TITLE
Add list of authors

### PR DESCRIPTION
**Pull request by louisjc**
Here is a proposal for #155. It implement a list of authors with the list of each contribution. The page is accessible under https://swiftweekly.github.io/authors/ but isn't referenced anywhere for now.

This commit does not add the bios referenced in the code. _Example using twitter bios:_

---

### CODE REVIEW

File: authors.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+layout: default
+title: List of authors
```
Comment (jessesquires) [inline authors.html]: let just call this "Authors"

File: authors.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+permalink: /authors/
```
Comment (jessesquires) [inline authors.html]: we can remove the `permalink:`

File: authors.html [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+---
+<h1 class="center">List of authors</h1>
+{% for author in site.data.authors %}
+  {% assign posts = site.posts | where: "author", author[0] %}
+  <h2 class="author-name"><a href="https://twitter.com/{{ author[1].twitter }}">{{ author[1].name }}</a> <small>{{ posts.size }} posts</small></h2>
+  <p>{{ author[1].bio }}</p>
+  <ul class="posts-list">
+    {% for post in posts %}
+    <li><a href="{{ post.url }}">{{ post.title }}</a></span>
```
Comment (jessesquires) [inline authors.html]: looks like a random `</span>` in here, need to replace with `</li>` 😄

File: css/style.css [hunk 4]
```diff
@@ -159,3 +159,22 @@ blockquote {
...
+  color: #333;
+  font-style: italic;
+  border-bottom: 1px dotted #333;
+}
+.author-name a:hover {
+  color: #fbaf40;
+  border-color: #fbaf40;
+}
+
+.posts-list li {
```
Comment (jessesquires) [inline css/style.css]: i don't think we need this with the fix above.

File: css/style.css [hunk 5]
```diff
@@ -159,3 +159,22 @@ blockquote {
...
+  font-size: 40px;
+  font-weight: lighter;
+  color: #333;
+  font-style: italic;
+  border-bottom: 1px dotted #333;
+}
+.author-name a:hover {
+  color: #fbaf40;
+  border-color: #fbaf40;
+}
```
Comment (jessesquires) [inline css/style.css]: can we remove these author styles and use existing ones? (like what's on a post)

**Comment by jessesquires** [comment]
After these changes, can you post a screenshot (without bios) ? 😄

**Comment by BasThomas** [comment]
This looks great! Thanks, @louisjc!

**Comment by louisjc** [comment]
I've made the changes! Here is the last version without bios:

---

capture d ecran 2017-01-26 a 10 08 52

File: authors.html [hunk 6]
```diff
@@ -0,0 +1,16 @@
+---
+layout: default
+title: Authors
+permalink: /authors/
+---
+<h1 class="center">List of authors</h1>
+{% for author in site.data.authors %}
+  {% assign posts = site.posts | where: "author", author[0] %}
+  <h2 class="lead post-title"><a href="https://twitter.com/{{ author[1].twitter }}"><em>{{ author[1].name }}</em></a> <small>{{ posts.size }} posts</small></h2>
```
Comment (BasThomas) [inline authors.html]: Nitpicking here, but can we conditionally show `post` / `posts` based on the amount of posts?

**Comment by louisjc** [comment]
@BasThomas done!

**Comment by jessesquires** [comment]
Thanks so much @louisjc ! 👍 🎉

**Pull request merged**